README.md

```diff
@@ -46,6 +46,15 @@ When run on a solution in order to analyze dependencies prior to upgrade, the to
 
 Extensions may be managed centrally for a project as described [here](docs/design/Extension_Management.md).
 
+### Experimental features
+
+Feature flags can be used to turn some experimental features on or off. Functionality behind feature flags may or may not be made public and are considered a way to test features that may be unstable. In order to enable a feature, set an environment variable to a semi-colon delimited list of feature names. For example: `$env:UA_FEATURES="FEATURE1;FEATURE2"`.
+
+Current features that are available to try out include:
+
+- `ANALYZE_OUTPUT_FORMAT`: Enables a `--format` flag for the `analyze` command that can take other options. Currently restricted to `sarif` and `html`
```

Comment:
@bleaphar this renames the feature flag so it's a bit more self-descriptive. Thoughts?